bind-ponyfill/bind-ponyfill-tests.ts

```diff
@@ -0,0 +1,12 @@
+import ponyBind = require('bind-ponyfill');
```

Comment:
Not sure why this file is showing up as added since it already exists on `types-2.0`, but I assume the added text on the bottom is a mistake.